rest-api/spring-rest/pom.xml

```diff
@@ -22,12 +22,61 @@
       <artifactId>common-rest</artifactId>
       <version>${project.parent.version}</version>
     </dependency>
+
+    <!-- common library dependencies -->
+    <dependency>
+      <groupId>uk.gov.gchq.gaffer</groupId>
+      <artifactId>time-library</artifactId>
+      <version>${project.parent.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>uk.gov.gchq.gaffer</groupId>
+      <artifactId>bitmap-library</artifactId>
+      <version>${project.parent.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>uk.gov.gchq.gaffer</groupId>
+      <artifactId>sketches-library</artifactId>
+      <version>${project.parent.version}</version>
+    </dependency>
     <dependency>
       <groupId>uk.gov.gchq.gaffer</groupId>
-      <artifactId>${store.type}-store</artifactId>
+      <artifactId>jcs-cache-service</artifactId>
       <version>${project.parent.version}</version>
     </dependency>
 
+    <!-- store dependencies -->
+    <dependency>
+      <groupId>uk.gov.gchq.gaffer</groupId>
+      <artifactId>map-store</artifactId>
+      <version>1.14.1-SNAPSHOT</version>
+    </dependency>
+    <dependency>
+      <groupId>uk.gov.gchq.gaffer</groupId>
+      <artifactId>federated-store</artifactId>
+      <version>1.14.1-SNAPSHOT</version>
+    </dependency>
+    <dependency>
+      <groupId>uk.gov.gchq.gaffer</groupId>
+      <artifactId>accumulo-store</artifactId>
+      <version>1.14.1-SNAPSHOT</version>
+    </dependency>
+    <dependency>
+      <groupId>uk.gov.gchq.gaffer</groupId>
+      <artifactId>proxy-store</artifactId>
+      <version>1.14.1-SNAPSHOT</version>
+    </dependency>
+    <dependency>
+      <groupId>uk.gov.gchq.gaffer</groupId>
+      <artifactId>hbase-store</artifactId>
+      <version>1.14.1-SNAPSHOT</version>
+    </dependency>
+    <dependency>
+      <groupId>uk.gov.gchq.gaffer</groupId>
+      <artifactId>parquet-store</artifactId>
+      <version>1.14.1-SNAPSHOT</version>
```

Comment:
```suggestion
      <version>${project.parent.version}</version>
```

Reply:
Version needs parameterising here and for the other store libraries.

Reply:
Yep good spot